telegram/_chat.py

```diff
@@ -184,7 +184,7 @@ class Chat(TelegramObject):
 
     def __init__(
         self,
-        id: int,
+        id: int,  # pylint: disable=invalid-name
```

Comment:
do you know why `redefined-builtin` is not needed here? o.0

Reply:
nope, was wondering the same thing

Reply:
ah it's because there's a pylint ignore at the top of the file

Reply:
ah okay 👍